### DISCUSSION TITLE
Will backrest resume an interrupted backup?

### DISCUSSION THREAD

**Discussion by caiotizio** [garethgeorge/backrest#472] [Q&A] [answered]
hi there,
as you can read [here](https://restic.readthedocs.io/en/latest/faq.html) restic support resume of an interrupted backup. is that feature also available in backrest?

**Comment by garethgeorge** [comment, marked as answer]
Hey, yep! This should be supported today out of the box. Backrest is using restic 0.17.0 as of writing which includes the resumability feature.

It should be mentioned that restic will still rescan all previously uploaded files, but shouldn't repack / reupload anything. It should be much faster. Note that an interrupted backup will still show as failed in Backrest's UI but the partial data will be there in the index (internal to your repo) and used by restic on the next attempt.
